$OpenBSD: patch-pkg-config-0_27_1_pkg_m4,v 1.1 2019/01/01 10:48:43 ajacoutot Exp $

Our pkg-config(1) supports --variable var package
(introduced in GNU's pkgconfig 0.28)

Index: pkg-config-0.27.1/pkg.m4
--- pkg-config-0.27.1/pkg.m4.orig
+++ pkg-config-0.27.1/pkg.m4
@@ -197,3 +197,20 @@ AC_SUBST([noarch_pkgconfigdir], [$with_noarch_pkgconfi
 m4_popdef([pkg_default])
 m4_popdef([pkg_description])
 ]) dnl PKG_NOARCH_INSTALLDIR
+
+
+dnl PKG_CHECK_VAR(VARIABLE, MODULE, CONFIG-VARIABLE,
+dnl [ACTION-IF-FOUND], [ACTION-IF-NOT-FOUND])
+dnl -------------------------------------------
+dnl Since: 0.28
+dnl
+dnl Retrieves the value of the pkg-config variable for the given module.
+AC_DEFUN([PKG_CHECK_VAR],
+[AC_REQUIRE([PKG_PROG_PKG_CONFIG])dnl
+AC_ARG_VAR([$1], [value of $3 for $2, overriding pkg-config])dnl
+
+_PKG_CONFIG([$1], [variable="][$3]["], [$2])
+AS_VAR_COPY([$1], [pkg_cv_][$1])
+
+AS_VAR_IF([$1], [""], [$5], [$4])dnl
+])dnl PKG_CHECK_VAR
